command/job_status.go

```diff
@@ -26,6 +26,20 @@ type JobStatusCommand struct {
 	evals     bool
 	allAllocs bool
 	verbose   bool
+	json      bool
+	tmpl      string
+}
+
+type JobJson struct {
+	Summary          *api.JobSummary
+	Allocations      []*api.AllocationListStub
+	LatestDeployment *api.Deployment
+	Evaluations      []*api.Evaluation
+}
+
+type jobIDNamespacePair struct {
+	ID        string
+	Namespace string
 }
```

Comment:
Upon seeing this struct and thinking "man, we must have this already, right?" I remembered we have `structs.NamespacedID` for just this purpose. Let's use that here.